### PR TITLE
Admin profile validation

### PR DESCRIPTION
**Pull request by ayushpatel1248**
## Issue that this pull request solves

- previously if we try ot update the admin profile username due to validation admin was not able to enter the numbers or _ in there validation 

- but now i have changed the validation and now admin can use number as well as _ in there username 

 Closes: #1042 

## Proposed changes

- in this i have changed the validation by changing the username schema where i have changed the regular expression for the username so that now admin can include numbers as well as the special _ type symbols in there username

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

https://github.com/HITK-TECH-Community/Community-Website/assets/127100604/421deec0-ae30-40af-a0d7-0a203c1089da

### CODE REVIEW

File: frontend/src/pages/Q&A/Q&A.jsx [hunk 1]
```diff
@@ -109,6 +114,7 @@ function Ques(props) {
           });
           setFormErrors({});
           setCheckedState(new Array(Tags.length).fill(false));
+          setButtonPressed(false);
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Q&A/Q&A.jsx:117]: Why is this change again coming?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@ayushpatel1248 there are so many unnecessary change in this file that its hard to follow, please clean it up and only keep the required changes

**Comment by ayushpatel1248** [comment]
@Kajol-Kumari  removed the unnecessary changes and kept only required changes , please check it out

**Review by Kajol-Kumari — APPROVED** [review]

**Comment by Kajol-Kumari** [comment]
@ayushpatel1248  i don' see the second part of the issue in this pr - https://github.com/HITK-TECH-Community/Community-Website/issues/1042#issuecomment-2163461523
